spec/models/course_spec.rb

```diff
@@ -24,12 +24,132 @@
 require 'rails_helper'
 
 RSpec.describe Course, type: :model do
-  subject { create(:course) }
-
+  let(:subject) { create(:course) }
   describe 'associations' do
     it { should belong_to(:provider) }
     it { should belong_to(:accrediting_provider) }
     it { should have_and_belong_to_many(:subjects) }
+    it { should have_many(:site_statuses) }
+    it { should have_many(:sites) }
+  end
+
+  describe `no site statuses` do
```

Comment:
Using backticks here is weird as it's also used to run inline shell commands, we should change (there are more in the rest of the file) these to single/double quotes.